vm/actor/src/builtin/miner/types.rs

```diff
@@ -9,105 +9,179 @@ use clock::ChainEpoch;
 use encoding::{serde_bytes, tuple::*};
 use fil_types::{PoStProof, RegisteredSealProof, SectorNumber};
 use num_bigint::bigint_ser;
-use vm::{DealID, TokenAmount};
+use vm::{DealID, Randomness, TokenAmount};
 
 pub type CronEvent = i64;
 pub const CRON_EVENT_WORKER_KEY_CHANGE: CronEvent = 1;
-pub const CRON_EVENT_PRE_COMMIT_EXPIRY: CronEvent = 2;
-pub const CRON_EVENT_PROVING_PERIOD: CronEvent = 3;
+pub const CRON_EVENT_PROVING_DEADLINE: CronEvent = 2;
+pub const CRON_EVENT_PROCESS_EARLY_TERMINATIONS: CronEvent = 3;
 
 /// Storage miner actor constructor params are defined here so the power actor can send them to the init actor
 /// to instantiate miners.
 #[derive(Serialize_tuple, Deserialize_tuple)]
 pub struct MinerConstructorParams {
```

Comment:
There is a duplicate of this in `power/types.rs` (go puts it there because there was a circular dep for them, and I only kept it there to not make a conflict with these changes). Can you just delete that other one, or have it alias this to future proof?